clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -1105,6 +1105,7 @@ void Clang::AddPreprocessingOptions(Compilation &C, const JobAction &JA,
                                     const InputInfo &Output,
                                     const InputInfoList &Inputs) const {
   const bool IsIAMCU = getToolChain().getTriple().isOSIAMCU();
+  const bool IsFPGA = Args.hasArg(options::OPT_fintelfpga);
```

Comment:
A minor renaming suggestion:
```suggestion
  const bool IsIntelFPGA = Args.hasArg(options::OPT_fintelfpga);
```